dockertest/images.py

```diff
@@ -641,13 +639,13 @@ def clean_all(self, fqins):
         else:
             preserve_fqins = []
         preserve_fqins.append(self.default_image)
-        preserve_fqins = set(preserve_fqins)
-        preserve_fqins.discard(None)
-        preserve_fqins.discard('')
+        preserve_fqins_set = set(preserve_fqins)
+        preserve_fqins_set.discard(None)
+        preserve_fqins_set.discard('')
         self.verbose = False
         try:
             for name in fqins:
-                if name in preserve_fqins:
```

Comment:
R0204:644,8: DockerImages.clean_all: Redefinition of preserve_fqins type from list to set